WebContent/js/components/JobInstance.js

```diff
@@ -34,6 +34,15 @@ class JobInstance extends React.Component {
         };
 
         this.handleKeyDown = this.handleKeyDown.bind(this);
+        this.checkForExpand = this.checkForExpand.bind(this);
+        setTimeout(this.checkForExpand, 1500); // Give time for UI code to render before toggling expand
```

Comment:
Note: This isn't solved by waiting for DidMount so I need a timeout

Reply:
Hi, could you please clarify, in what specific case the componentDidMount does not help? Is it app2app related case? 
I have tried to trigger the checkForExpand function from a componentDidMount, and it worked with hardcoded expand = true; for me.